src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessThreadCollection.cs

```diff
@@ -5,21 +5,16 @@
 
 namespace System.Diagnostics
 {
-    /// <devdoc>
-    ///    <para>[To be supplied.]</para>
```

Comment:
`[To be supplied]` ✔️  😆

Reply:
Those comments have been there since .NET Framework: https://referencesource.microsoft.com/#System/services/monitoring/system/diagnosticts/ProcessThreadCollection.cs,15